### ISSUE TITLE
Udemy Discount

### ISSUE THREAD

**Issue by th0mas** [dwyl/learn-flutter#62]
For the next 12 hours, Udemy has discounted their Flutter course to £13.99

https://www.udemy.com/course/the-complete-flutter-app-development-course-for-android-ios/

**Comment by nelsonic** [comment]
Yeah, that's [Udemy](https://github.com/dwyl/learn-elixir/issues/56#issuecomment-324316686) being [Udemy](https://github.com/dwyl/learn-flutter/issues/33) ... 🙄 
And if you view that link using a VPN through India/Philippines on Windows instead of Mac, it's even _cheaper_.  
The "ticking clock discount" is the oldest trick in the book. "_Act now before it's too late_". 🙄 
![image](https://user-images.githubusercontent.com/194400/89187201-de51cb80-d594-11ea-9841-855a9529f0c1.png)
https://www.clickz.com/how-countdown-timers-can-be-used-to-drive-ecommerce-sales/109133/

On my Mac it's €13.99 with a 5 hour clock:
![image](https://user-images.githubusercontent.com/194400/89187352-135e1e00-d595-11ea-9af2-43df02a3e21d.png)

On Windows it's €10.99 and has a 23 hour clock:

![udemy-flutter-10 99](https://user-images.githubusercontent.com/194400/89190473-6df97900-d599-11ea-82a0-d5f72f37a3cc.PNG)

Bottom line: don't trust their mega discount count timer _ever_. 
But if the course seems like good content, get it. 👍 

Crucially the page says the course was last updated in **November 2018**, that's a _long_ time in Flutter dev. 
Most of the basics are still probably relevant, but if you've tried running older (1yr+) Flutter code, there are occasionally breaking changes https://flutter.dev/docs/release/breaking-changes that can be frustrating when learning. 

I'd probably go with "_The Complete 2020 Flutter Development Bootcamp with Dart Officially created in collaboration with the Google Flutter team_" by Angela Yu of https://www.appbrewery.co last updated **April 2020**:
https://www.udemy.com/course/flutter-bootcamp-with-dart
![image](https://user-images.githubusercontent.com/194400/89190914-0ee83400-d59a-11ea-966b-e60127b101b2.png)

Again, ignore the ticking clock and just get the course that looks best.

I recently discovered https://www.filledstacks.com/ that has a _lot_ of high quality and ***FREE*** Flutter content:
![image](https://user-images.githubusercontent.com/194400/89191300-a483c380-d59a-11ea-9569-868eda5f1f41.png)

There's a slack channel to ask questions and all the code is available free on GitHub: https://github.com/FilledStacks/flutter-tutorials
